HopsanGUI/Widgets/AnimationWidget.cpp

```diff
@@ -129,8 +129,7 @@ AnimationWidget::AnimationWidget(QWidget *parent) :
     mpTimeSlider = new QSlider(Qt::Horizontal);
 
     mpSpeedSpinBox = new QDoubleSpinBox(this);
-    mpSpeedSpinBox->setMinimum(-20.0);
-    mpSpeedSpinBox->setMaximum(20.0);
+    mpSpeedSpinBox->setRange(std::numeric_limits<double>::min(),std::numeric_limits<double>::max());
```

Comment:
This prevents negative values, is that intended?
std::numeric_limits<double>::min() is a positive very small value
Otherwise maybe the setRange call can be completely removed

But if the reason is to prevent 0 (and negative scaling) then OK

Reply:
Aha, just looked at the issue, for realtime, I guess you do not want negative values.
What about playbak, is that an other spinbox? (did not check code) 
Because pause and reverse could be useful then